Woof/Woof/ViewModel/SitterMainTabViewModel.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+
+/// The view model for a pet sitter main tab view, responsible for preparing and processing data for the view.
+final class SitterMainTabViewModel: ObservableObject {
+    /// Resets the current sitter role in the app by default value `none`.
```

Comment:
?
```suggestion
    /// Resets the current user role in the app to the default value.
```

Reply:
Thanks for the good suggestion. Fixed it.